android/engine/src/test/java/org/smartregister/fhircore/engine/util/extension/PatientExtensionTest.kt

```diff
@@ -255,9 +255,12 @@ class PatientExtensionTest : RobolectricTest() {
   }
 
   @Test
-  fun testExtractAgeShouldReturnCallGetAgeStringFromDaysWhenPatientHasBirthDate() {
-    val calendar =
-      Calendar.getInstance().apply { timeInMillis = (timeInMillis - (1L * 365 * 24 * 3600 * 1000)) }
+  fun testExtractAgeShouldReturnAgeStringFromDaysWhenPatientHasBirthDate() {
+    val calendar = Calendar.getInstance()
+    calendar.apply {
+      timeInMillis =
+        (timeInMillis - (1L * calendar.getActualMaximum(Calendar.DAY_OF_YEAR) * 24 * 3600 * 1000))
```

Comment:
I think this fix from @marklosh looks cleaner to me, fewer special numbers :D, https://github.com/opensrp/fhircore/pull/3116/files#diff-f7b77a5722cb99a28fd93c5bab900a5b20dfc103e66d798be9ccb8194c5c64f4R265

Reply:
Agreed, reverting this